src/turbine/runner/baserunner.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, path_to_data_app: str, name: str = None):
     def app_config(self):
         config = {}
         try:
-            with open(os.path.abspath(f"{self.path_to_data_app}") + "/app.json") as fd:
+            with open(os.path.abspath(f"{self.path_to_data_app}/app.json")) as fd:
```

Comment:
@james won't it make sense to do this?
```suggestion
            with open(os.path.abspath(os.path.join(self.path_to_data_app, "app.json"))) as fd:
```

Reply:
actually yeah this is cleaner.